src/backend/db/database_manager.py

```diff
@@ -64,14 +64,15 @@ def add_customer(self):
         session = DatabaseManager._session_factory()
         try:
             logger.debug(f"Creating database: {customer_db_name}")
-            create_db_query = f"CREATE DATABASE `{customer_db_name}`"
+            create_db_query = f"CREATE DATABASE IF NOT EXISTS `{customer_db_name}`"
```

Comment:
Since it is add customer let's remove if not exists command

Reply:
modified sir